src/views/Pools/components/PoolTabButtons.tsx

```diff
@@ -1,66 +1,65 @@
+  const { t } = useTranslation()
 
-const StyledLink = styled(UiKitLink)`
-  width: 100%;
+  const viewModeToggle = <ToggleView viewMode={viewMode} onToggle={(mode: ViewMode) => setViewMode(mode)} />
 
-  &:hover {
-    text-decoration: none;
-  }
-`
+  const liveOrFinishedSwitch = (
+    <ButtonMenu activeIndex={isExact ? 0 : 1} scale="sm" variant="subtle">
+      <ButtonMenuItem as={Link} to={`${url}`}>
+        {t('Live')}
+      </ButtonMenuItem>
+      <NotificationDot show={hasStakeInFinishedPools}>
+        <ButtonMenuItem as={Link} to={`${url}/history`}>
+          {t('Finished')}
+        </ButtonMenuItem>
+      </NotificationDot>
+    </ButtonMenu>
+  )
 
-const PoolTabButtons = ({ stakedOnly, setStakedOnly, hasStakeInFinishedPools }) => {
-  const { url, isExact } = useRouteMatch()
-  const { t } = useTranslation()
+  const stakedOnlySwitch = (
+    <Flex mt={['4px', null, 0, null]} ml={[0, null, '24px', null]} justifyContent="center" alignItems="center">
+      <Toggle scale="sm" checked={stakedOnly} onChange={() => setStakedOnly((prev) => !prev)} />
+      <Text ml={['4px', '4px', '8px']}>{t('Staked only')}</Text>
+    </Flex>
+  )
 
-  return (
-    <Flex alignItems="center" justifyContent="center" mb="32px">
-      <Flex alignItems="center" flexDirection={['column', null, 'row', null]}>
-        <ButtonMenu activeIndex={isExact ? 0 : 1} scale="sm" variant="subtle">
-          <ButtonMenuItem as={Link} to={`${url}`}>
-            {t('Live')}
-          </ButtonMenuItem>
-          <NotificationDot show={hasStakeInFinishedPools}>
-            <ButtonMenuItem as={Link} to={`${url}/history`}>
-              {t('Finished')}
-            </ButtonMenuItem>
-          </NotificationDot>
-        </ButtonMenu>
-        <Flex mt={['4px', null, 0, null]} ml={[0, null, '24px', null]} justifyContent="center" alignItems="center">
-          <Toggle scale="sm" checked={stakedOnly} onChange={() => setStakedOnly((prev) => !prev)} />
-          <Text ml="8px">{t('Staked only')}</Text>
+  if (isXs) {
+    return (
+      <Flex flexDirection="column" alignItems="flex-start" mb="24px">
+        {viewModeToggle}
+        <Flex width="100%" justifyContent="space-between">
+          {liveOrFinishedSwitch}
+          {stakedOnlySwitch}
         </Flex>
       </Flex>
-      <Flex ml="24px" alignItems="center" justifyContent="flex-end">
-        <StyledLink external href="https://docs.pancakeswap.finance/syrup-pools/syrup-pool">
-          <Button px={['14px', null, null, null, '20px']} variant="subtle">
-            <ButtonText color="backgroundAlt" bold fontSize="16px">
-              {t('Help')}
-            </ButtonText>
-            <HelpIcon color="backgroundAlt" ml={[null, null, null, 0, '6px']} />
-          </Button>
-        </StyledLink>
-      </Flex>
+    )
+  }
+
+  return (
+    <Flex
+      alignItems="center"
+      justifyContent={['space-around', 'space-around', 'flex-start']}
+      mb={['24px', '24px', '24px', '0px']}
```

Comment:
Just an FYI, I see in other spots you are using `null`. Using `null` is preferable as it doesn't create a new style.

Reply:
Good to know 👍